src/sentry/integrations/bitbucket/client.py

```diff
@@ -70,7 +73,7 @@ def get_issues(self, repo):
     def create_issue(self, repo, data):
         return self.post(path=BitbucketAPIPath.issues.format(repo=repo), data=data)
 
-    def search_issues(self, repo, query):
+    def search_issues(self, repo: str, query: str) -> dict[str, Any]:
```

Comment:
Should it be `str` twice?
```suggestion
    def search_issues(self, repo: str, query: str) -> dict[str, str]:
```

Reply:
this returns a json blob -- I don't know the shape of it unfortunately but it's not `dict[str, str]`